scripts/functions.sh

```diff
@@ -187,3 +187,31 @@ function configure_apt_sources {
   curl --silent --fail ${HOST_RCBOPS_REPO}/apt-mirror/rcbops-release-signing-key.asc | apt-key add -
 
 }
+
+function safe_to_replace_artifacts {
+
+  # The mainline branch should always be one version ahead of the rc branch,
+  # as the next rc branch will be cut from mainline. This function checks
+  # the current branch against the rc branch and returns 1 if it is the same
```

Comment:
First phrase is useful. 

"This function checks the current branch against the rc branch and returns 1 if it is the same" is basically the code, so I doubt it's useful.

I'd replace this part of the comment with:
"A return of 0 means it's safe to replace artifacts. This should happen when there is no rc branch, or the versions in the two branches differ, to avoid overlaps."